ddcz/views.py

```diff
@@ -474,12 +476,24 @@ def users_list(request):
 @require_http_methods(["GET"])
 def user_profile(request, user_profile_id, nick_slug):
     user_profile = get_object_or_404(UserProfile, id=user_profile_id)
+    try:
+        creations = Author.objects.get(user=user_profile).get_all_creations()
+    except Author.DoesNotExist as e:
+        creations = False
+
+    description = LEVEL_DESCRIPTIONS["0"]
+    if user_profile.level in LEVEL_DESCRIPTIONS.keys():
```

Comment:
fwiw, you can do just `if user_profile.level in LEVEL_DESCRIPTIONS:`

Reply:
Nice, thank you